package.json

```diff
@@ -6,23 +6,22 @@
   "author": "StackStorm <support@stackstorm.com>",
   "description": "A simple helpful robot for your Company, with StackStorm Support",
   "dependencies": {
-    "hubot"             : "^2.19.0",
+    "hubot"             : "^3.1.1",
     "hubot-diagnostics" : "0.0.2",
     "hubot-redis-brain" : "0.0.4",
     "hubot-scripts"     : "^2.17.2",
     "hubot-help"        : "^0.2.2",
-    "hubot-stackstorm"  : "^0.8.0",
-    "hubot-flowdock"    : "^0.7.6",
-    "hubot-matteruser"  : "^3.6.0",
-    "hubot-yammer"      : "git+https://github.com/athieriot/hubot-yammer",
-    "hubot-spark"       : "^1.7.0",
+    "hubot-stackstorm"  : "git+https://github.com/StackStorm/hubot-stackstorm#update-components",
```

Comment:
Just a reminder that to pin to specific `hubot-stackstorm` released version (`v0.9.0`) before merging this PR.
Eg. Merge `hubot-stackstorm`, release and then `st2chatops`